src/application.rs

```diff
@@ -27,6 +27,7 @@ pub enum OsdTypes {
 	SourceVolumeMuteToggle = 7,
 	BrightnessRaise = 8,
 	BrightnessLower = 9,
+	MaxVolume = 10,
```

Comment:
We should change the enum name to something like "ArgTypes" or anything similar

Reply:
I don't quite get what you mean? Could you explain in more detail?

Reply:
I just meant changing the whole enum name (from "OsdTypes" to something else)

Reply:
Done